ICRPG extended/ICRPGextended.html

```diff
@@ -0,0 +1,524 @@
+              <p align="center">
+                <input type="number" name="attr_fury" value="0" >
+             </p>
+        </th>
+    </div>
+
+</div>
+<br />
+
+<h3 style="text-align: center; color: #FFF; background-color: #c00607;">Magic Details</h3>
+<input type="checkbox" class="sheet-hider" name="attr_Magic_details"><span></span></span>
+ <div>
+  <div id="Magic">
+    <br />
+   <div class='sheet-3colrow'>
+     <div class='sheet-col'>
+         <th>
+             <label>Blood</label>          
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <br />
+             Level Known <input type="number" name="attr_blood" value="0">
+         </th>
+         <br/>
+         <br/>
+         <th>
+             <label>Curse</label>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <br />
+             Level Known <input type="number" name="attr_Curse" value="0">
+         </th>
+         <br/>
+         <br/>
+         <th>
+             <label>Demon</label>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <br />
+             Level Known <input type="number" name="attr_Demon" value="0">
+         </th>
+         <br/>
+         <br/>
+         <th>
+             <label>Dimension</label>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <br />
+             Level Known <input type="number" name="attr_Dimension" value="0">
+         </th>
+         <br/>
+         <br/>
+     </div>
+     <div class='sheet-col'>
+         <th>
+             <label>Elder</label>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <br />
+             Level Known <input type="number" name="attr_Elder" value="0">
+         </th>
+         <br/>
+         <br/>
+         <th>
+             <label>Energy</label>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <br />
+             Level Known <input type="number" name="attr_Energy" value="0">
+         </th>
+         <br/>
+         <br/>
+         <th>
+             <label>Fire</label>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <br />
+             Level Known <input type="number" name="attr_Fire" value="0">
+         </th>
+         <br/>
+         <br/>
+         <th>
```

Comment:
`<th>` is tag that belongs in a table. They will need to be removed. What are they being used for here?

Reply:
> `<th>` is tag that belongs in a table. They will need to be removed. What are they being used for here?

Like I said before, I learned html especially for this project and I took a lot of tags from example sheets to learn it. Anyway, the tag is removed and was apparently not doing anything for the sheet. I hope the sheet is finally ready to upload now!